tools/sgx-lkl-setup

```diff
@@ -25,9 +25,11 @@ sudo iptables -t nat -A POSTROUTING -s 10.0.1.0/24 ! -d 10.0.1.0/24 -j MASQUERAD
 
 echo "Done"
 
-echo "Setting up Linux kernel module to support userspace FSGSBase..."
-sudo insmod ${SGXLKL_ROOT}/tools/kmod-set-fsgsbase/mod_set_cr4_fsgsbase.ko val=1
-sudo rmmod mod_set_cr4_fsgsbase
-dmesg | tail -n 10
-echo "Done"
+if [[ -f ${SGXLKL_ROOT}/tools/kmod-set-fsgsbase ]]; then
```

Comment:
Wouldn't it be helpful to also output a message if the module has not been loaded, so the the user knows that SGX-LKL will either require a suitable installed kernel or emulate userspace FSGSBASE with a performance overhead?

Reply:
Well, then you need to do the check to see if the module was loaded, e.g. by the DKMS package. I don't want to complicate this more. SGX-LKL already outputs a message if FSGSBASE is emulated.

Reply:
Ok, I was just thinking about outputting a message that no module was loaded.

Reply:
It's already documented on how to install and enable the fsgsbase dkms package when using the binary packages, that should be enough. For building from source it will be loaded.